examples/wasm-demo-cairo1/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+mod utils;
+
+use cairo1_run::{cairo_run_program, Cairo1RunConfig};
+use cairo_lang_sierra::ProgramParser;
+use cairo_vm::types::layout_name::LayoutName;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+extern "C" {
+    #[wasm_bindgen(js_namespace = console)]
+    fn log(msg: &str);
+}
+
+#[wasm_bindgen(start)]
+pub fn start() {
+    crate::utils::set_panic_hook();
+}
+
+// TODO: check why this is needed. Seems wasm-bindgen expects us to use
+// `std::error::Error` even if it's not yet in `core`
+macro_rules! wrap_error {
+    ($xp: expr) => {
+        $xp.map_err(|e| JsError::new(e.to_string().as_str()))
+    };
+}
+
+#[wasm_bindgen(js_name = runCairoProgram)]
+pub fn run_cairo_program() -> Result<String, JsError> {
+    let cairo_run_config = Cairo1RunConfig {
+        layout: LayoutName::all_cairo,
+        relocate_mem: true,
+        trace_enabled: true,
+        serialize_output: true,
+        ..Default::default()
+    };
+
+    // using cairo-lang 1.1.1 and ../caigo-programs/cairo-1-programs/bitwise.cairo
```

Comment:
This isn't the program that is being used. It's actually `examples/wasm-demo-cairo1/bitwise.sierra`